src/Groups/group_characters.jl

```diff
@@ -651,6 +814,75 @@ function decomposition_matrix(modtbl::GAPGroupCharacterTable)
     return matrix(ZZ, GAP.Globals.DecompositionMatrix(modtbl.GAPTable))
 end
 
+@doc Markdown.doc"""
+    class_multiplication_coefficient(::Type{T} = fmpz, tbl::GAPGroupCharacterTable, i::Int, j::Int, k::Int) where T <: IntegerUnion
+
+Return the class multiplication coefficient of the classes `i`, `j`, and `k`
+of the group ``G`` with ordinary character table `tbl`,
+as an instance of `T`.
+
+The class multiplication coefficient ``c_{i,j,k}`` of the classes
+``i, j, k`` equals the  number of pairs ``(x, y)`` of elements ``x, y \in G``
+such that ``x`` lies in class ``i``, ``y`` lies in class ``j``,
+and their product ``xy`` is a fixed element of class ``k``.
+
+In the center of the group algebra of ``G``, these numbers are found as
+coefficients of the decomposition of the product of two class sums ``K_i``
+and ``K_j`` into class sums:
+```math
+K_i K_j = \sum_k c_{ijk} K_k.
+```
+Given the character table of a finite group ``G``,
+whose classes are ``C_1, \ldots, C_r`` with representatives ``g_i \in C_i``,
+the class multiplication coefficient ``c_{ijk}`` can be computed
+with the following formula:
+```math
+    c_{ijk} = |C_i| |C_j| / |G|
+              \sum_{\chi \in Irr(G)} \chi(g_i) \chi(g_j) \chi(g_k^{-1})
+              / \chi(1).
+```
+On the other hand the knowledge of the class multiplication coefficients
+admits the computation of the irreducible characters of ``G``.
+
+# Examples
+```jldoctest
+julia> class_multiplication_coefficient(character_table("A5"), 2, 3, 4)
+5
+
+julia> class_multiplication_coefficient(character_table("A5"), 2, 4, 4)
+0
+
+```
+"""
+function class_multiplication_coefficient(::Type{T}, tbl::GAPGroupCharacterTable, i::Int, j::Int, k::Int) where T <: IntegerUnion
+  return T(GAP.Globals.ClassMultiplicationCoefficient(tbl.GAPTable, i, j, k)::GAP.Obj)
+end
+
+class_multiplication_coefficient(tbl::GAPGroupCharacterTable, i::Int, j::Int, k::Int) = class_multiplication_coefficient(fmpz, tbl, i, j, k)
+
+@doc Markdown.doc"""
+    possible_class_fusions(subtbl::GAPGroupCharacterTable, tbl::GAPGroupCharacterTable)
+
+Return the array of possible class fusions from `subtbl` to `tbl`.
+Each entry is an array of positive integers, where the value at position `i`
+is the position of the conjugacy class in `tbl` that contains the `i`-th class
+of `subtbl`.
+
+# Examples
+```jldoctest
+julia> possible_class_fusions(character_table("A5"), character_table("A6"))
+4-element Vector{Vector{Int64}}:
+ [1, 2, 3, 6, 7]
+ [1, 2, 3, 7, 6]
+ [1, 2, 4, 6, 7]
+ [1, 2, 4, 7, 6]
+
+```
+"""
+function possible_class_fusions(subtbl::GAPGroupCharacterTable, tbl::GAPGroupCharacterTable)
+  fus = GAP.Globals.PossibleClassFusions(subtbl.GAPTable, tbl.GAPTable)::GAP.GapObj
+  return [Vector{Int}(x) for x in fus]
```

Comment:
This is a minor detail, but Julia won't know the type of `x`; it seems to me it is always a GAP list here. If so, a type assertion will allow Julia to deduce which `Vector{Int}` constructor is invoked during precompile time, and hence also deduce the return type of this function (i.e. make it type stable)
```suggestion
  return [Vector{Int}(x::GapObj) for x in fus]
```

Reply:
Yes.